ansible.cfg

```diff
@@ -13,11 +13,13 @@ any_errors_fatal_setup = true
 any_errors_fatal_experiments = false
 
 
+
 # TODO [nku] can I control this cfg from the makefile? `make run suite=abc id=new` vs `make run-v suite=abc id=new` vs `make run-vvvv suite=abc id=new`
 stdout_callback = community.general.selective
 
+
 # speedup by using ssh pipelining
 pipelining = True
-[ssh_connection]
-pipelining = True
-ssh_args = -o ControlMaster=auto -o ControlPersist=1200
\ No newline at end of file
+#[ssh_connection]
```

Comment:
Is this disabled on purpose?

Reply:
I don't remember why I commented this out.
I'm bringing it back.